lib/init-input.js

```diff
@@ -1,7 +1,8 @@
 /* global config, prompt */
 var base = require('@deck/base')
 var extend = require('extend-object')
-var exec = require('child_process').execSync
+var execSync = require('child_process').execSync
+var exec = require('child_process').exec
```

Comment:
Where is the exec var being referenced? Isn't this an extraneous assignment? I would have thought the linter would have caught that